src/less/filter-button/base/filter-button.less

```diff
@@ -0,0 +1,67 @@
+button.filter-button,
+a.filter-link {
+    .filter-button-base;
+
+    background-color: @filter-button-background-color;
+
+    &:focus,
+    &:hover,
+    &:active {
+        background-color: @filter-button-hover-background-color;
+    }
+}
+
+.filter-button__cell,
+.filter-link__cell {
+    .filter-button-cell;
+}
+
+a.filter-link:visited {
+    .filter-link-visited;
+
+    color: @filter-button-color;
+}
+
+button.filter-button[aria-pressed="true"],
+a.filter-link--selected {
+    .filter-button-selected;
+
+    background-color: @filter-button-selected-background-color; // should we just lighten @color-b4;
+    color: @filter-button-selected-color;
+
+    &:focus,
+    &:hover,
+    &:active {
+        background-color: @filter-button-selected-hover-background-color; // should we just lighten @color-b4;
+    }
+}
+
+button.filter-button[disabled],
```

Comment:
This section of styles for button are the exact same as the anchor tag ones. Is this intentional? Should we combine them?

Reply:
Good callout. Fixing.